gradle/plugins/src/main/kotlin/com.hedera.gradle.lifecycle.gradle.kts

```diff
@@ -23,12 +23,6 @@ plugins {
 // tasks from the 'build' group
 defaultTasks("tasks")
 
-tasks.named<TaskReportTask>("tasks") {
-    if (!isDetail) {
-        displayGroup = "build"
-    }
-}
-
```

Comment:
When I added this, the idea was to only show the most important tasks when users do use CLI and run `./gradlew`, but this does not work as expected. `idDetail` is always false, even if you run with `--detail`. Lets remove this cusotmization. If you use Gradle through the IDE it does not matter at all.